### PR TITLE
Use uuid instead of space.id

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/7305

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by kulmann — APPROVED** [review]

File: changelog/unreleased/bugfix-access-details-shown-for-all-space-members [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Bugfix: Access details shown for all space members
```
Comment (kulmann) [inline changelog/unreleased/bugfix-access-details-shown-for-all-space-members:1]: Wait, this PR doesn't need its own changelog item. Could you please remove it and add the PR link to the other changelog item where you introduced the share details drop?

**Review by kulmann — CHANGES_REQUESTED** [review]

File: changelog/unreleased/bugfix-access-details-shown-for-all-space-members (hunk 1 above)
Comment (JanAckermann) [reply changelog/unreleased/bugfix-access-details-shown-for-all-space-members:1]: sure

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap [hunk 2]
```diff
@@ -6,10 +6,10 @@ exports[`Collaborator ListItem component share inheritance indicators show when
     <avatar-image-stub userid="brian" user-name="Brian Murphy" width="48" class="files-collaborators-collaborator-indicator"></avatar-image-stub>
     <div class="oc-text-truncate oc-width-1-1">
       <div class="oc-flex oc-flex-middle"><span class="oc-text-truncate oc-m-rm"><span aria-hidden="true" class="files-collaborators-collaborator-name">Brian Murphy</span></span> <span class="oc-invisible-sr">Share receiver name: Brian Murphy (brian@owncloud.com)</span>
-        <oc-button-stub id="share-access-details-toggle-asdf" appearance="raw" class="oc-ml-xs files-collaborators-collaborator-access-details-button">
+        <oc-button-stub id="share-access-details-toggle-557aadea-f98a-4e4a-bb52-66c374f8e22d" appearance="raw" class="oc-ml-xs files-collaborators-collaborator-access-details-button">
```
Comment (kulmann) [inline packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap:9]: random uuids in snapshot tests will not work 😓

Comment (JanAckermann) [reply packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap:9]: on it

**Review by kulmann — APPROVED** [review]

File: packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap (hunk 2 above)
Comment (kulmann) [reply packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.js.snap]: 👍

**Pull request merged**